.github/vale/dicts/aiven.dic

```diff
@@ -28,6 +29,7 @@ CloudWatch
 codecs
 compliances
 Conduktor
+Console
```

Comment:
I don't think this is necessary: Aiven is already present and Console is a correct word